app/views/user_mailer/password_reset.html.erb

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Views are embedded inside of the application layout (`app/views/layouts/application.html.erb`) which is responsible for specifying the doctype, head, body, and html elements. View templates that aren't layouts should only need the content inside the body tag.